************* Module asyncio.locks
/log/firmware/firmware/usr/lib/python3.8/asyncio/locks.py:83:4: W0212: Access to a protected member _is_coroutine of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/asyncio/locks.py:83:29: W0212: Access to a protected member _is_coroutine of a client class (protected-access)
/log/firmware/firmware/usr/lib/python3.8/asyncio/locks.py:94:15: E1101: Instance of '_ContextManager' has no '__await__' member (no-member)
/log/firmware/firmware/usr/lib/python3.8/asyncio/locks.py:102:4: C0103: Argument name "tb" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/asyncio/locks.py:404:4: C0103: Argument name "n" doesn't conform to snake_case naming style (invalid-name)
/log/firmware/firmware/usr/lib/python3.8/asyncio/locks.py:10:0: W0611: Unused import futures (unused-import)

-----------------------------------
Your code has been rated at 9.56/10

